packages/target-tauri/runtime-tauri/runtime.ts

```diff
@@ -331,6 +331,27 @@ class TauriRuntime implements Runtime {
     }
     return ''
   }
+  transformStickerURL(sticker_path: string): string {
+    const matches = sticker_path.match(/.*(:?\\|\/)(.+?)\1stickers\1(.*)/)
```

Comment:
would/should this fix https://github.com/deltachat/deltachat-desktop/issues/4548 ?
At least in my test it didn't.
In electron runtime a return sticker_path.replace(/%20/g, ' ') fixed the problem (decodeURI did not)

Reply:
the whole premise of https://github.com/deltachat/deltachat-desktop/issues/4548 is weird IMO, you normally don't do folders that have uri encoded names. I believe adb should fix his bot, there is no real reason for having this.

Reply:
> should this fix

no, otherwise I'd have linked it or mentioned it.